platform/viewer/cypress/integration/study-list/OHIFStudyList.spec.js

```diff
@@ -21,31 +21,42 @@ describe('OHIF Study List', function() {
       //cy.get('@modalities')
     });
 
-    it('searches Patient Name with exact string', function() {
+    it('Displays several studies initially', function () {
+      cy.waitStudyList();
+      cy.get('@searchResult2').should($list => {
+        expect($list.length).to.be.greaterThan(1);
+        expect($list).to.contain('Juno');
+        expect($list).to.contain('832040');
+      });
+    });
+
+    it('searches Patient Name with exact string', function () {
       cy.get('@PatientName').type('Juno');
       //Wait result list to be displayed
       cy.waitStudyList();
-      cy.get('@searchResult').should($list => {
+      cy.get('@searchResult2').should($list => {
         expect($list.length).to.be.eq(1);
         expect($list).to.contain('Juno');
       });
     });
 
-    it('searches MRN with exact string', function() {
+    /* TODO - re-enable this once the test server is fixed to allow searching by mrn
```

Comment:
The MRN query is commented out because querying by the hex code doesn't work in the test server setup.
This ALSO means that the actual results come back incorrectly.  Hmm, let me add a tweak to this to allow it to work.

Reply:
Makes sense